src/index.html

```diff
@@ -5,8 +5,404 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <a href="index.html" class="head__logo">
+        <img src="images/logo.png" alt="The logo of Moyo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list_li">
+            <a href="#" class="is-active nav__list_link">Apple</a>
+          </li>
+
+          <li class="nav__list_li">
+            <a href="#" class="nav__list_link">Samsung</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link">Samsung</a>
```
rename classes following BEM